source/developers/remote-assets.rst

```diff
@@ -8,11 +8,11 @@
 Remote Assets
 -------------
 
-Remote assets are binary files, typically images, videos, pdf documents, etc. which are hosted outside of Crafter Studio.  Remote assets could be hosted in AWS, Box or some other server accessed through WebDAV, CMIS. etc.
+Remote assets are binary files, typically images, videos, pdf documents, etc. which are hosted outside of Crafter Studio and Crafter Delivery.  Remote assets could be hosted in AWS, Box or some other server accessed through WebDAV, CMIS. etc.
 
-Various data sources are available to help manage/use assets hosted outside of Crafter Studio in your site.  The ``Developer`` section contains some examples on how to store assets remotely, such as :ref:`use-s3-to-store-assets` and :ref:`use-box-to-store-assets`.  The ``Site Administrators`` section contains information on how to configure Crafter Studio to access services used for storing assets remotely here: :ref:`studio-configuration`.
+For authoring, various Crafter Studio data sources are available to help manage/select assets hosted outside of your site.  The ``Developer`` section contains some examples on how to store assets remotely, such as :ref:`use-s3-to-store-assets` and :ref:`use-box-to-store-assets`.  The ``Site Administrators`` section contains information on how to configure Crafter Studio to access services used for storing assets remotely here: :ref:`studio-configuration`.
 
-Crafter Studio by default has the remote assets controller open up the remote repository for read access via the URL pattern ``/remote-assets/STORE-TYPE/PROFILE-ID/PATH-TO-ASSET``, where:
+Browser access to remote assets on your site is provided by Crafter Delivery remote assets controller via the URL pattern ``/remote-assets/STORE-TYPE/PROFILE-ID/PATH-TO-ASSET``, where:
```

Comment:
The remote assets controller is provided by Crafter Engine, which is in both authoring and delivery environments.  Authoring contains Studio, Engine and Deployer. Studio only uploads the binary files to the external storage (aws/webdav/box) and then Engine does the render (for preview) & Deployer the indexing.

So, instead of `Crafter Delivery`, use `Crafter Engine` :
`Browser access to remote assets on your site is provided by Crafter Engine remote assets controller ...`

Reply:
That should work, as long as we don't call it a Studio feature as before.